src/ingest_validation_tools/directory-schemas/mxif-v0.yaml

```diff
@@ -1,5 +1,31 @@
 files:
+  # Level0/
   -
-    pattern: 'TODO'
-    description: 'Directory structure not yet specified.'
-    required: True
\ No newline at end of file
+    pattern: '(Level0|level0)_raw\/cyc_[\d]+\.(tif|tiff)'
+    description: 'MXiF raw tile data by cycle and channel.'
+    required: True
+    is_qa_qc: True
+  -
+    pattern: '(Level0|level0)_lab_pre-processing\/cyc_[\d]+\/tile_layout\.json'
```

Comment:
This tile layout information is not lab pre-processing and is raw data. It tells us where each tile lives in the space of the full image prior to stitching. I think these files should live in the same directory as the tile data. I suggest (regex might not be perfect below):

pattern: '(Level0|level0)_raw\/cyc_[\d]+\tile_layout\.json'

This means that each tile layout information file is stored next to the tile images.